Mastodon/Diffable/Search/SearchHistorySection.swift

```diff
@@ -32,7 +32,12 @@ extension SearchHistorySection {
                 guard let user = item.object(in: context.managedObjectContext) else { return }
                 cell.configure(
                     me: authContext.mastodonAuthenticationBox.authenticationRecord.object(in: context.managedObjectContext)?.user,
-                    viewModel: .init(value: user, followedUsers: authContext.mastodonAuthenticationBox.inMemoryCache.$followingUserIds.eraseToAnyPublisher(), blockedUsers: authContext.mastodonAuthenticationBox.inMemoryCache.$blockedUserIds.eraseToAnyPublisher()),
+                    viewModel: SearchHistoryUserCollectionViewCell.ViewModel(
+                        value: user,
+                        followedUsers: authContext.mastodonAuthenticationBox.inMemoryCache.$followingUserIds.eraseToAnyPublisher(),
+                        blockedUsers: authContext.mastodonAuthenticationBox.inMemoryCache.$blockedUserIds.eraseToAnyPublisher(),
+                        followRequestedUsers: authContext.mastodonAuthenticationBox.inMemoryCache.$followRequestedUserIDs.eraseToAnyPublisher()
```

Comment:
We's also need to fetch the pending requests from https://docs.joinmastodon.org/methods/follow_requests/ to pre-seed this e.g. on user switch etc. basically the same as done here: https://github.com/mastodon/mastodon-ios/blob/develop/MastodonSDK/Sources/MastodonCore/Service/AuthenticationService.swift#L50

Reply:
Ah, you're right! I'll add that :)